<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>490</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cell Absorption</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="topWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="QWidget" name="midWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="midLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="midLWidget" native="true">
           <layout class="QVBoxLayout" name="midLLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="colLabel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(250, 250, 250);
border: 1px solid rgb(132, 136, 145);
border-bottom-color: transparent;
font: 11pt &quot;맑은 고딕&quot;;</string>
              </property>
              <property name="text">
               <string>Column</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="tabletTracking">
               <bool>false</bool>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(240, 240, 240);</string>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::CopyAction</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="arrowLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>170</number>
           </property>
           <property name="bottomMargin">
            <number>170</number>
           </property>
           <item>
            <widget class="QPushButton" name="leftArrow">
             <property name="styleSheet">
              <string notr="true">border: None;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>img/right_to_left_arrow.png</normaloff>img/right_to_left_arrow.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rightArrow">
             <property name="styleSheet">
              <string notr="true">border: None;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>img/left_to_right_arrow.png</normaloff>img/left_to_right_arrow.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="midRWidget" native="true">
           <layout class="QVBoxLayout" name="midRLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="slctColLabel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(250, 250, 250);
border: 1px solid rgb(132, 136, 145);
border-bottom-color: transparent;
font: 11pt &quot;맑은 고딕&quot;;</string>
              </property>
              <property name="text">
               <string>Selected Column</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="slctListWidget">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(240, 240, 240);</string>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::CopyAction</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="maximumSize">
         <size>
          <width>323</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {background-color: white; color: black; selection-background-color: white; selection-color: black;}
QComboBox QAbstractItemView {
	background-color: gray; color: white;
	selection-background-color: lightgray; selection-color: black}</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
border-left-color: transparent;
border-right-color: transparent;
border-bottom-color: transparent;</string>
     </property>
     <layout class="QGridLayout" name="bottomLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>13</number>
      </property>
      <property name="rightMargin">
       <number>13</number>
      </property>
      <property name="bottomMargin">
       <number>13</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 225, 225);
border: 1px solid rgb(180, 180, 180);
margin-left: 10px;
padding-left: 35px;
padding-right: 35px;
padding-top: 5px;
padding-bottom: 5px;</string>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
